float64 Ux_mps
float64 Uy_mps
float64 r_radps
float64 ax_mps2
float64 ay_mps2
float64 speed_rpm
float64 delta_rad
float64 Single_Motor_TxR_Nm
float64 latitude
float64 longitude
float64 altitude
float64 roll_rad
float64 pitch_rad
float64 yaw_rad
int32 gps_state
float64 UWB_x_m
float64 UWB_y_m
float64 EKF_x_m
float64 EKF_y_m